build.gradle.kts

```diff
@@ -155,16 +159,17 @@ spotless {
         target("**/*.java")
         removeUnusedImports()
         googleJavaFormat()
-        targetExclude("**/generated/**", "**/vendor/**")
+        targetExclude("**/generated/**", "**/vendor/**", "sentry-spring-jakarta/**", "sentry-spring-boot-starter-jakarta/**")
     }
 
     kotlin {
         target("**/*.kt")
-        ktlint()
+        ktlint("0.47.1")
```

Comment:
Why do we need to pin the version here?
If its because of compatibility, add a comment and move the version to the Config.kt file.

Reply:
I think we need it for the new API that allows us to configure some details. Will check if it's still needed.